client/src/components/QA/QACardAnswers.jsx

```diff
@@ -32,6 +30,18 @@ const QACardAnswers = ({ currentAnswers }) => {
     setToggle(true);
   }, []);
 
+  const APICall = (answerId) => {
```

Comment:
These function names could probably be more specific to your module/api endpoint.

Reply:
Will update these